src/refinement-list/refinement-list.ts

```diff
@@ -30,7 +30,10 @@ export type RefinementListState = {
   selector: 'ais-refinement-list',
   template: `
     <div
-      [class]="cx()"
+      [ngClass]="[
+        cx(),
+        state.items.length === 0 ? cx('', 'noRefinement') : ''
```

Comment:
there's `state.canRefine` for this no?

Reply:
We should rely on the connector's `canRefine` value when available.

Reply:
That's what Vue IS is doing but as explained in the description of this PR, I don't think it's good idea to pick canRefine for 2 reasons:
1. it will introduce a inconsistency with Vanilla behaviour, which I want to avoid. (I prefer being inconsistent with Vue IS 😄 )
2. canRefine is true when searching for facet values => noRefinement class is never applied while searching for facet values

Reply:
Ideally I wish there's `noRefinement` value coming from the connector, so that all the flavors can just use it.
I agree with the consistency with vanilla, too. (of course, it has to be consistent everywhere, though)